compiler/luci-interpreter/pal/mcu/PALFill.h

```diff
@@ -0,0 +1,22 @@
+/*
+* Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
Did you copy paste this?

```suggestion
* Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```

Reply:
I am also have concerns that we need this pal in "big" luci-interpreter